nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -1043,6 +1043,47 @@ public void restrictivelyAnnotatedMethodsWorkWithNullnessFromDataflow2() {
         .doTest();
   }
 
+  @Test
+  public void OverridingRestrictivelyAnnotatedMethod() {
+    compilationHelper
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:UnannotatedSubPackages=com.uber.lib.unannotated",
+                "-XepOpt:NullAway:AcknowledgeRestrictiveAnnotations=true"))
+        .addSourceLines(
+            "Test.java",
```

Comment:
Suggestion: Rename this to `TestNegativeCases.java` and the class accordingly. Would make the test easier to read.

Reply:
done